src/pages/task/actionButtons/previousCommits/index.tsx

```diff
@@ -51,204 +36,138 @@ export const PreviousCommits: React.FC<PreviousCommitsProps> = ({ taskId }) => {
     variables: { taskId },
   });
 
+  const { baseTask, buildVariant, displayName, versionMetadata } =
+    taskData?.task ?? {};
+  const { order: skipOrderNumber, projectIdentifier } =
+    versionMetadata?.baseVersion ?? {};
+  const bvOptionsBase = {
+    tasks: [applyStrictRegex(displayName)],
+    variants: [applyStrictRegex(buildVariant)],
+  };
+
   // We don't error for this query because it is the default query that is run when the page loads.
   // If it errors it probably means there is no base version, which is fine.
```

Comment:
I think you can delete this comment since there's a `skip` on this now :0 (doesn't run by default)